Example/YelpAPI.xcodeproj/project.pbxproj

```diff
@@ -124,6 +128,8 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				0B5080851CBEEA950071508E /* OHHTTPStubs.framework in Frameworks */,
+				0B5080831CBEEA800071508E /* OCMock.framework in Frameworks */,
```

Comment:
Or we could tackle that removal in a followup, it's not crucial here (and I'm pretty excited to get Travis enabled)